src/commands/deploy.ts

```diff
@@ -189,10 +190,17 @@ export async function deploy(context: IDeployWizardContext, confirmDeployment: b
 
     // Don't wait
     validateWebSite(correlationId, node).then(
-        () => {
+        async () => {
             // ignore
         },
-        () => {
-            // ignore
+        async () => {
+            if (!node || !(await node.root.client.getSiteConfig()).linuxFxVersion) {
+                // this currently only works for Linux apps, so don't delay and just exit if it's a Windows app
+                return;
+            }
+            // it can take over 10 minutes for the detectors to appear
```

Comment:
Well, I have a delay before I even start checking, and then check it for 10 more minutes (because who knows when the heck it'll up ready to view.)  I'm fine with polling for it more often, but it just seemed pointless to start polling because I know it's gonna be a while.

Reply:
Well that's my question - does it take _at least_ 10 minutes? That's different than "it can take over 10 minutes".  If so I'm fine with this delay but I would clarify the comment in the code.